File: Data Structures For Interview/Python/Matrix Programs/OddEven.py
def oddeven(list1):

    even=0
    odd=0

    for i in range(len(list1)):
        for j in range(len(list1)):
            if list1[i][j]%2==0:
                even=even+1
            else:
                odd=odd+1
    
    print("Frequency of even numbers :- ",even)
    print("Frequency of odd numbers :- ",odd)

list1=[
    [1,2,3],
    [4,5,6],
    [7,8,9]
]

oddeven(list1)

File: Data Structures For Interview/JavaScript/Matrix Programs/OddEven.js
function oddeven(m1){

    let even=0;
    let odd=0;

    for(let i=0;i<m1.length;i++){
        for(let j=0;j<m1[0].length;j++){
            if(m1[i][j]%2==0){
                even++;
            }
            else{
                odd++;
            }
        }
    }
    
    console.log("Frequency of even numbers are :- "+even);
    console.log("Frequency of odd numbers are :- "+odd);

}

let m1=[
    [1,2,3],
    [4,5,6],
    [7,8,9]
];

let m2=[
    [1,2,3],
    [4,5,6],
    [7,8,9]
];

oddeven(m1);